### PR TITLE
Select applicants

### PR DESCRIPTION
**Pull request by juopmu**

### CODE REVIEW

File: app/controllers/applications_controller.rb [hunk 1]
```diff
@@ -25,6 +25,11 @@ def index
     end
   end
 
+  def select
+    @event.applications.update_all(["selected = (id IN (?))", params[:selected_ids]])
+    redirect_to event_applications_path(@event), notice: "Cool! Changes saved."
+  end
```
Comment (porras) [inline app/controllers/applications_controller.rb]: Please don't forget to make sure that this action is [only available to admins](https://github.com/rubymonsters/rgbapp/blob/1d7e0727fa269dbefa3dc9a78aaf14fa7fc36cff/app/controllers/applications_controller.rb#L3).

File: config/routes.rb [hunk 2]
```diff
@@ -4,5 +4,7 @@
     resources :applications
   end
 
+  put "/events/:event_id/applications", to: "applications#select"
+
```
Comment (porras) [inline config/routes.rb]: A shorter and maybe simpler way to add this is having just `put :applications, to: "applications#select"`, but inside the `resources :events` block (everything inside gets prepended with `/events/:event_id/`).

**Comment by porras** [comment]
This is great stuff!

File: config/routes.rb (hunk 2 above)
Comment (juopmu) [reply config/routes.rb]: Done.

**Comment by porras** [comment]
👏 👏 👏 👏 👏 👏

**Pull request merged**